doc/source/turkey.md

```diff
@@ -0,0 +1,33 @@
+
+The health is thought to be one of the areas where Turkey is good, but the numbers are saying this is not true. In 2018, the total expenditure on health as a share of GDP was the lowest among OECD countries at %6.3 of GDP. Also, Turkey has one of the highest rates of obesity in the world (%29.5).
+
+Due to the fact that the majority of the Turkey’s population is Muslim, there is a common bias about Turkey that is it is an Islamic country, but it is not. Since its foundation, Republic of Turkey is a secular state with no official state religion. The first article of the Turkish constitution says that the country’s regime is republic, and the next article says that Turkey is a secular, social, democratic, nationalist, state of law and it respects human rights. Even though there have always been governments against some of these articles, including the current one, and they have taken some actions to executes their aims, they were not successful, until this one.
+
+Turkey was founded by the old soldiers of Ottoman Empire. Mustafa Kemal who was later given the surname of “Atatürk” (father of the Turks) was the leader of the independence movement of which aim was to defend Anatolia against enemies in the time following the First World War. They refused the conditions of the Treaty of Sevres and fought for years. Finally, they were able to establish Turkey with its today’s borders except Hatay which joined to Turkey on their will after they gained their independence from France.
+
+Turkey is considered to be a European country thanks to the Treaty of Lausanne with which Turkey’s independence and sovereignty were accepted. But it is not a member of the European Union. Turkey has applied to become a member of EU and some of the legislations and the laws has been adapted to those of the EU but there were a few points on which the two sides were not able to agree. So, the process has not been completed and the Turkish government is not trying to join the EU anymore.
+
+If you are talking about a country, you should mention its culture. So, let’s dive into Turkish culture. As we have mentioned above, there are more than one nation in Turkey, and this was also the case in the Ottoman Empire, the predecessor of Turkey. Therefore, the culture of the country is also a mix of different cultures. Like its geographical location, Turkey’s culture is also a bridge between Europe and Middle East. The cultures in the area were affected so much from each other that there are some disputed meals about their origins. For example, “yoğurt”, a globally known food, is claimed by Greek people. They have been so successful defending their claims that it is known as Greek yogurt almost everywhere. There are so many meals like yogurt, but we will not further discuss about that matter in this article.
```

Comment:
If you are talking about a country, you should mention its culture. So, let’s dive into Turkish culture. As we have mentioned above, there is more than one nation in Turkey, and this was also the case in the Ottoman Empire, the predecessor of Turkey. Therefore, the culture of the country is also a mix of different cultures. Like its geographical location, Turkey’s culture is also a bridge between Europe and the Middle East. The cultures in the area were affected so much by each other that there are some disputed meals about their origins. For example, “yoğurt”, a globally known food, is claimed by Greek people. They have been so successful in defending their claims that it is known as Greek yogurt almost everywhere. There are so many meals like yogurt, but we will not further discuss that matter in this article.